cloud_functions/api_router/utils/cloud_utils.py

```
@@ -0,0 +1,13 @@
+"""Cloud utils for api_router."""
+import google.auth
+import google.auth.transport.requests
+
+
+def get_oauth_token() -> str:
+    """Retrieve 0auth2 token."""
+    credentials, _ = google.auth.default(
+        scopes=["https://www.googleapis.com/auth/cloud-platform"]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)